samples/csharp/MicrosoftEduGraphSamples/Workflows/AssignmentWorkflow.cs

```diff
@@ -123,5 +123,41 @@ public async Task AssignmentsEvolvableEnumsAsync()
                 Console.WriteLine($"AssignmentsEvolvableEnumsAsync: {ex.ToString()}");
             }
         }
-    }
+
+        /// <summary>
+        /// Workflow to get assignments from all the classes which are not archived
+        /// </summary>
+        /// <param name="isTeacher">True value accepts Teacher account and false for Student account</param> 
+        public async Task CreateAndPatchAssignment(bool appOnly = false)
+        {
+            try
+            {
+                string assignmentId = string.Empty;
+
+                // Get a Graph client using delegated permissions
```

Comment:
// Get a Graph client based on the appOnly parameter

Reply:
Changed to "// Get a Graph client using appOnly Parameter"